docs/hpc/applications/easybuild.md

```diff
@@ -38,195 +198,249 @@ archspec/0.1.4-GCCcore-11.3.0                            SCOTCH/7.0.1-iimpi-2022
+        buildenv/default-intel-2023a
+        buildenv/default-intel-2023b
 
-For example to load GROMACS 2021.3 (with CUDA acceleration) from the production GPU software stack and use within a batch you, you could do the following:
+```
 
-Using production GPU modules within a batch script
+Evidently, the latest version for the `foss` compiler is `buildenv/default-foss-2023b`. But what compiler is that actually? The quickest way is simply loading the module and see what else has been loaded:
 
-```bash
-#PBS -lwalltime=01:00:00
-#PBS -l select=1:ncpus=4:mem=24gb:ngpus=1:gpu_type=RTX6000
+```console
+$ ml buildenv/default-foss-2023b
+§ ml
+```
 
-module purge
-module load tools/prod
-module load GROMACS/2021.3-foss-2021a-CUDA-11.3.1
+These modules are loaded per default:
 
-gmx <your options>
+```console
+Currently Loaded Modules:
+  1) tools/prod                        14) libfabric/1.19.0-GCCcore-13.2.0
+  2) GCCcore/13.2.0                    15) PMIx/4.2.6-GCCcore-13.2.0
+  3) zlib/1.2.13-GCCcore-13.2.0        16) UCC/1.2.0-GCCcore-13.2.0
+  4) binutils/2.40-GCCcore-13.2.0      17) OpenMPI/4.1.6-GCC-13.2.0
+  5) GCC/13.2.0                        18) OpenBLAS/0.3.24-GCC-13.2.0
+  6) numactl/2.0.16-GCCcore-13.2.0     19) FlexiBLAS/3.3.1-GCC-13.2.0
+  7) XZ/5.4.4-GCCcore-13.2.0           20) FFTW/3.3.10-GCC-13.2.0
+  8) libxml2/2.11.5-GCCcore-13.2.0     21) gompi/2023b
+  9) libpciaccess/0.17-GCCcore-13.2.0  22) FFTW.MPI/3.3.10-gompi-2023b
+ 10) hwloc/2.9.2-GCCcore-13.2.0        23) ScaLAPACK/2.2.0-gompi-2023b-fb
+ 11) OpenSSL/1.1                       24) foss/2023b
+ 12) libevent/2.1.12-GCCcore-13.2.0    25) buildenv/default-foss-2023b
 ```
 
-More information on running GPU jobs may be found on our [GPU Jobs page](../queues/gpu-jobs.md)
+As evident, quite some modules are being loaded and thus the build environment has the following software packages loaded:
 
-## User Software Development
-In order to make it easier for those users who are either developing software, or need to quickly install some software to test, we are now offering a development module: 'tools/eb-dev'.
+- GCC
+- OpenSSL
+- OpenMPI
+- OpenBLAS/FlexiBLAS
+- FFTW
+- ScaLAPACK
 
-This allows you to install software with EasyBuild in your own user space, so that you can use a locally modified version of software, or set up a reproducible build environment when working on your own software development projects. The latter will allow you to change your build environment without going through Anaconda or any other virtual environment.
-If you only want to switch compilers etc, you can make use of the already installed "buildenv" modules which will give you convenient access to various versions of both GCC and Intel compilers, next to MPI access.
+next to some auxiliary programs we don't look too much in here. This would be a very basic build environment. For the sake of this example. we require `cmake`. Thus, we are search for it and load the required module:
```

Comment:
```suggestion
next to some auxiliary programs we don't look too much in here. This would be a very basic build environment. For the sake of this example. we require `cmake`. Thus, we search for it and load the required module:
```